### PR TITLE
GameDB: Force English in Tomb Raider - The Angel of Darkness US

### PR DESCRIPTION
**Pull request by pgert**

### CODE REVIEW

**Comment by Mrlinkwii** [comment]
its a game bug not an emulation bug :/

**Comment by F0bes** [comment]
I'm not opposed to this PR, makes it better for users and it's not like we have a better way to fix it.

**Comment by refractionpcsx2** [comment]
I agree, I'm fine with this patch, it is a stupid mistake by the devs, and we won't be able to fix it any better way.

**Comment by refractionpcsx2** [comment]
With the second comment line, can you get rid of the extra space at the beginning? thanks

**Comment by RedDevilus** [comment]
The game devs surely sniffed some glue when they were coding for that series, like it's a different developer for Tomb Raider Legend.

File: bin/resources/GameIndex.yaml [hunk 1]
```diff
@@ -42474,6 +42474,10 @@ SLUS-20467:
         // Fix ingame SPS by interchanging vclipw.xyz vf5, vf5w with cfc2.
         patch=1,EE,001d90e0,word,48489000
         patch=1,EE,001d90e4,word,4bc529ff
+        author=Agrippa
```
Comment (MrCK1) [inline bin/resources/GameIndex.yaml:42477]: Comment should read

 // Force English - else the game crashes with French or Spanish set
 //  in the BIOS configuration - a game bug, not an emulator bug.

**Comment by pgert** [comment]
Can't merge the commits (squash).
Perhaps this can be "sponsored"?

**Comment by refractionpcsx2** [comment]
yeah sure, I can do that

**Comment by pgert** [comment]
Thank you.

**Pull request merged**